include/eve/arch/cpu/logical_wide.hpp

```diff
@@ -128,6 +128,10 @@ namespace eve
     EVE_FORCEINLINE explicit logical(S v) noexcept
                   : storage_base(detail::make(eve::as<logical>{}, v)) {}
 
+    //! Construct from a `bool`
+    EVE_FORCEINLINE explicit logical(bool v) noexcept
```

Comment:
Using `convertible_to<bool>` requirements instead of having 2 overlaods leads to ambiguity with the Generator constructor as lambda are convertible to bool.